src/main/java/com/google/opensesame/servlets/UserServlet.java

```diff
@@ -87,4 +87,19 @@ public PreparedQuery queryInDatabase(
   }
 
   // TODO: make error handling conform with Richie's error handling
+  /**
+   * Check if a user is stored with userID in Datastore. Invalid return false.
+   *
+   * @param userID
+   * @return true if userID exists in datastore, false otherwise.
+   */
+  public Boolean hasProfile(String userID) {
+    try {
+      Key userKey = KeyFactory.createKey(PersonBuilder.ENTITY_NAME, userID);
+      datastore.get(userKey);
+    } catch (Exception e) {
```

Comment:
Again, if we could be a bit more specific about which exceptions we're catching here, that would be great.